pom.xml

```diff
@@ -324,6 +324,88 @@
       <properties>
         <maven.compiler.release>8</maven.compiler.release>
       </properties>
+      <build>
+        <pluginManagement>
+          <plugins>
+            <plugin>
+              <groupId>org.eclipse.m2e</groupId>
```

Comment:
Can we keep this Eclipse stuff in a separate patch?
It deserves separate discussion

Reply:
Sure thing. I'll update the PR.

Reply:
Updated.